Package.swift

```diff
@@ -1,19 +1,28 @@
-//    Copyright 2016 Swift Studies
-//
-//    Licensed under the Apache License, Version 2.0 (the "License");
-//    you may not use this file except in compliance with the License.
-//    You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-//    Unless required by applicable law or agreed to in writing, software
-//    distributed under the License is distributed on an "AS IS" BASIS,
-//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//    See the License for the specific language governing permissions and
-//    limitations under the License.
```

Comment:
Please do not remove the license/copywrite headers any files on this repo.

Reply:
Thank you for your comment!
I'll add license/copywrite.